### PR TITLE
Or 221 trigger percy on label using matrices

### PR DESCRIPTION
**Pull request by frshwtr**

### CODE REVIEW

File: apps/website/_tutorials/new-component/create-a-new-component-part-7.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Now run `npm run test -w components/o-example`. You should see our new tests are
 
 To check all component demos for any visual bugs that may have been introduced accidentally as part of a change may be rather taxing as a manual piece of work. To help, demos may be run through [percy.io](https://percy.io/) to highlight visual differences between two versions of a component automatically.
 
-We can't run Percy yet as we haven't released a version of our component to compare changes against. But later, when we have released our component, [percy.io](https://percy.io/) will run automatically against new pull requests.
+We can't run Percy yet as we haven't released a version of our component to compare changes against. But later, when we have released our component, [percy.io](https://percy.io/) we can run Percy tests by applying `Run Percy` label on your pull request.
```
Comment (akomiqaia) [inline apps/website/_tutorials/new-component/create-a-new-component-part-7.md:125]:
```suggestion
We can't run Percy yet as we haven't released a version of our component to compare changes against. But later, when we have released our component, [percy.io](https://percy.io/) we can run Percy tests by applying `Percy` label on your pull request.
```

**Comment by akomiqaia** [comment]
@frshwtr Does this PR still need to open?

**Comment by frshwtr** [comment]
> @frshwtr Does this PR still need to open?

I moved it to draft, thanks for pointing it out :)

I made this as an illustrative PR. Whether we do it or not is another question.

**Comment by notlee** [comment]
Closing as we're unlikely to work further on Percy optimisations.

**Pull request closed without merging**